spec/views/identities/index.html.haml_spec.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'identities/index', type: :view do
+  describe 'identities view' do
+    let(:identity) { FactoryBot.create :identity }
+    it 'lists a number of identities' do
```

Comment:
i don't think this actually tests the view. It looks to be making an instance of the model and testing that.
here's an example of a view spec (copied from rates rebates)
```
# frozen_string_literal: true

require 'rails_helper'

RSpec.describe 'admin/rebate_forms/index', type: :view do
  let(:council) { FactoryBot.create(:council) }

  before do
    FactoryBot.create_list(:rebate_form, 2)
    assign(:council, council)
    assign(:rebate_forms, RebateForm.all.page(0))
    assign(:years, %w[2018 2019])
    render
  end

  it { expect(rendered).to include council.name }
  it { expect(rendered).to include '2018' }
  it { expect(rendered).to include '2019' }
end
```

Reply:
It needs to call render, to render the view -- and then ater that it checks that the expected things were rendered